docs/releases/updating_ez_platform.md

```diff
@@ -250,15 +250,15 @@ index 49a17a9..80c4cd7 100644
 The web assets must be dumped again if you are using the `prod` environment. In `dev` this happens automatically:
 
 ``` bash
-php app/console assetic:dump -e=prod
+php bin/console assetic:dump -e=prod
 ```
 
 If you encounter problems, additionally clear the cache and install assets:
 
 ``` bash
-php app/console cache:clear -e=prod
-php app/console assets:install --symlink -e=prod
-php app/console assetic:dump -e=prod
+php bin/console cache:clear -e=prod
+php bin/console assets:install --symlink -e=prod
```

Comment:
The symlinks are helpful in the development mode. On production it might be an option, not a must.

Reply:
Is this something that concerns v2 only, or v1 as well?

Reply:
I think both.

Reply:
If it's both then it will be fixed when the update guide is reviewed for 1.x